psen_scan_v2/OutputPinID pin_id
string name
bool state
